src/main/java/com/github/technus/tectech/loader/recipe/ResearchStationAssemblyLine.java

```diff
@@ -2572,7 +2572,7 @@ private void addEOHRecipes() {
                             FluidUtils.getFluidStack("molten.metastable oganesson", 144 * 256 * 4),
                             FluidUtils.getFluidStack("molten.shirabon", 144 * 256 * 4), },
                     CustomItemList.Machine_Multi_EyeOfHarmony.get(1),
-                    400 * MINUTES,
+                    1600 * MINUTES,
```

Comment:
regardless if this is merged or not, write the duration properly: 400 mins is 6h40mins, so in code it's `6*HOURS+40*MINUTES`, 1600mins is 26h40 mins. 

Having such a time of craft is stupid, increase voltage tier and divide the time. it's stupid to put such high time when its purpose is to be OC-ed

Reply:
I changed it to how you wanted it written. If the voltage is increased to uxv then you cannot oc the recipe until t7 eoh.

Reply:
this indicates a game design flaw. One more reason to keep it to what it is currently

Reply:
I just find it strange that the controller has a longer recipe time than its later casings. The aal is voltage locked to prevent tier skipping and uxv is gated by the eoh, so I think umv voltage is ideal.

Reply:
given how pricey the multi is as a whole ,energy is basically free so it makes no diff for a player to wait a bit less just for the controller. And at least we don't reach stupid base values.

Reply:
By that logic shouldn't the time on eoh casings be reduced as well?

Reply:
Energy and materials are both much easier at that stage of the game, which means this change is nothing but pure tedium. Our development is meant to move away from tedium, not to add more of it.

Reply:
yes but some people are against it so it's a hot topic within the team, and since the vision of the very endgame is blurry at best, we aren't decided to tweak stuff. We have yet to decide if we want to tone down power creep, time gates, etc. So this PR will likely not make it for the time being.

Reply:
I think meepmopper is trying to prove that the casings time are unjustly long and showing its hypocrisy

Reply:
That would make the actual PR pointless and a waste of time compared to changing the casings themselves. On top of that, this is a balance change in unclear territory and stable is coming up, so this is best delayed.

Reply:
@meepmopper You got a good idea for what casing times should be?

Reply:
> Energy and materials are both much easier at that stage of the game, which means this change is nothing but pure tedium. Our development is meant to move away from tedium, not to add more of it.

If that is the case then why is your new qft recipe 200 seconds long?

> Energy and materials are both much easier at that stage of the game, which means this change is nothing but pure tedium. Our development is meant to move away from tedium, not to add more of it.

If that is the case then why is your new qft recipe 200 seconds long?



> yes but some people are against it so it's a hot topic within the team, and since the vision of the very endgame is blurry at best, we aren't decided to tweak stuff. We have yet to decide if we want to tone down power creep, time gates, etc. So this PR will likely not make it for the time being.

Very few controllers are needed compared to the amount of eoh casings for each multi and gate. It doesn't make sense that making the controller having a long recipe time is harmful and only makes players wait, but it's fine that the casings are extremely large time sinks.

Reply:
QFT is unrelated to this and that recipe is not yet useful, so it's not an important balance situation like this one.

Reply:
as said, we need to clean up what should happen in our vision of the endgame. Crafting time and power consumption will naturally come after. I'm closing this PR for now, we will eventually reopen it if needed.